tests/testJointcal_cfht.py

```diff
@@ -0,0 +1,57 @@
+# See COPYRIGHT file at the top of the source tree.
+
+from __future__ import division, absolute_import, print_function
+
+import unittest
+import os
+
+from lsst.afw import geom, coord
+import lsst.utils
+import lsst.pex.exceptions
+
+import jointcalTestBase
+
+try:
+    data_dir = lsst.utils.getPackageDir('validation_data_jointcal')
+    os.environ['ASTROMETRY_NET_DATA_DIR'] = os.path.join(data_dir, 'cfht_and_index')
+except lsst.pex.exceptions.NotFoundError:
+    data_dir = None
+
+# We don't want the absolute astrometry to become significantly worse
+# than the single-epoch astrometry (about 0.040").
+absolute_error = 48e-3/jointcalTestBase.arcsec_per_radian
+# Set to True for a comparison plot and some diagnostic numbers.
+doPlot = False
+
+
+# for MemoryTestCase
+def setup_module(module):
+    lsst.utils.tests.init()
+
+
+class JointcalTestCFHT(jointcalTestBase.JointcalTestBase, lsst.utils.tests.TestCase):
+    def setUp(self):
+        jointcalTestBase.JointcalTestBase.setUp(self)
+        self.doPlot = doPlot
+        self.matchRadius = 0.1*geom.arcseconds
```

Comment:
It's in the separate tests to allow easier individual configuration, even if they're likely to have the same value most (all?) of the time.